src/codewhisperer/explorer/codewhispererNode.ts

```diff
@@ -67,14 +67,13 @@ export class CodeWhispererNode implements RootNode {
     public getChildren() {
         const autoTriggerEnabled =
             globals.context.globalState.get<boolean>(CodeWhispererConstants.autoTriggerEnabledKey) || false
-
+        if (AuthUtil.instance.isConnectionExpired()) {
+            return [createReconnectNode(), createLearnMore()]
+        }
         if (!AuthUtil.instance.isConnected()) {
             return [createSsoSignIn(), createLearnMore()]
         }
```

Comment:
This is a user-facing change worth mentioning in changelog. Something like:

> CodeWhisperer: show "Reconnect" instead of "Start" for expired connections, so that reconnecting is easier